MastodonSDK/Sources/MastodonCore/Extension/CoreDataStack/Status+Property.swift

```diff
@@ -53,16 +53,29 @@ extension Mastodon.Entity.Status {
         let attachments = mediaAttachments.compactMap { media -> MastodonAttachment? in
             guard let kind = media.attachmentKind,
                   let meta = media.meta,
-                  let original = meta.original,
-                  let width = original.width,       // audio has width/height
-                  let height = original.height
+                  let original = meta.original
             else { return nil }
-            
+
+            let size: CGSize
+            if let width = original.width,
+               let height = original.height {
+                size = CGSize(width: width, height: height)
+                // audio may have the width/height on the “small” (thumbnail?) property instead
+            } else if
+                kind == .audio,
+                let small = meta.small,
+                let width = small.width,
+                let height = small.height {
+                size = CGSize(width: width, height: height)
+            } else {
+                return nil
+            }
```

Comment:
This doesn't really help, if `meta.small` is nil.

```
(lldb) po media.remoteURL
▿ Optional<String>
  - some : "https://cdn.masto.host/mastodonart/media_attachments/files/109/302/787/076/266/355/original/04519903aafd999c.mp3"

(lldb) po meta
▿ Meta
  ▿ original : Optional<Format>
    ▿ some : Format
      - width : nil
      - height : nil
      - size : nil
      - aspect : nil
      - frameRate : nil
      ▿ duration : Optional<Double>
        - some : 205.688163
      ▿ bitrate : Optional<Int>
        - some : 162203
  - small : nil
  ▿ focus : Optional<Focus>
    ▿ some : Focus
      - x : 0.0
      - y : 0.0
  - length : nil
  - duration : nil
  - fps : nil
  - size : nil
  - width : nil
  - height : nil
  - aspect : nil
  - audioEncode : nil
  - audioBitrate : nil
  - audioChannels : nil
```

It's this [toot](https://mastodon.art/@simonstalenhag/109302790092203385) from the issue.